src/battle/overlay_12_0224E4FC.c

```diff
@@ -6473,155 +6477,155 @@ void ov12_022582B8(BattleSystem *bsys, BattleContext *ctx, int var, int battlerI
     }
 }
 
-static const int ov12_0226CDCC[145] = {
-    0,
-    18,
-    22,
-    25,
-    27,
-    31,
-    47,
-    37,
-    14,
-    55,
-    56,
-    48,
-    13,
-    58,
-    63,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    64,
-    66,
-    85,
-    86,
-    93,
-    119,
-    115,
-    130,
-    138,
-    147,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    12,
-    44,
-    142,
-    149,
-    150,
-    148,
-    151,
-    152,
-    24,
-    33,
-    34,
-    35,
-    43,
-    45,
-    46,
-    49,
-    52,
-    54,
-    62,
-    67,
-    68,
-    70,
-    73,
-    77,
-    78,
-    79,
-    80,
-    81,
-    82,
-    84,
-    87,
-    88,
-    89,
-    91,
-    92,
-    95,
-    96,
-    97,
-    126,
-    100,
-    101,
-    103,
-    105,
-    106,
-    109,
-    112,
-    113,
-    114,
-    120,
-    122,
-    123,
-    124,
-    125,
-    127,
-    128,
-    129,
-    131,
-    132,
-    134,
-    135,
-    140,
-    141,
-    143,
-    145,
-    154,
-    155,
-    156,
-    158,
-    159,
-    160,
-    161,
-    162,
-    163,
-    164,
-    165,
-    166,
-    167,
-    168,
-    170,
-    171,
-    173,
-    175,
-    218,
-    219,
-    220,
-    226,
-    246,
-    247,
-    248,
-    249,
-    260,
-    261,
-    118
+static const int sMoveStatusChangeScripts[145] = {
```

Comment:
some of these consts have spaces after them, also could 145 be a constant?

Reply:
no it's just a length